build/linux/installer/conf/fluent-bit-common.conf

```diff
@@ -85,10 +85,27 @@
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
 
+# TODO: Merge this with oms_output plugin as both of them are using the same plugin
```

Comment:
1. Did you check what would be perf difference running in same plugin or different plugin? 2. Did you check what's the additional impact on existing log collection with these new plugins?

Reply:
1. running it in same plugin is better because otherwise we will have duplicate data flowing in fluent-bit system
2. I did not see any large difference since again this is only once per minute. Will add a longer time high scale log test to investigate further